bin/lint

```diff
@@ -9,3 +9,4 @@ if [ -z "$*" ]
 then
    bin/brakeman --quiet --no-summary --no-pager
 fi
+terraform fmt -recursive
```

Comment:
Currently this linting script allows passing file paths as arguments to only lint those files, is it possible to include this for Terraform formatting like the others?